src/main/java/Duke.java

```diff
@@ -1,10 +1,115 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+    public static final String FRONT_SPACING = " ";
+    public static final String NEW_LINE = "\n";
+    public static final String TOP_LINE = "____________________________________________________________\n";
+    public static final String BOTTOM_LINE = "\n____________________________________________________________";
+    public static final String LIST_HEADER = FRONT_SPACING + "Here are the tasks in your list:\n";
+    public static final String INVALID_TASK = FRONT_SPACING + "Invalid task to insert";
+    public static final String TASK_HEADER = FRONT_SPACING + "Got it. I've added this task: \n";
+    public static final String TASK_FOOTER_FIRST_PART = FRONT_SPACING + "Now you have ";
+    public static final String TASK_FOOTER_SECOND_PART = " tasks in the list.";
+    public static final String INVALID_TASK_NUMBER = "Invalid task number";
+    public static final String MARKED_DONE_HEADER = "Nice! I've marked this task as done:\n";
+    public static final String GREETING_HEADER_TOP = FRONT_SPACING + "Hello! I'm Duke\n";
+    public static final String GREETING_HEADER_BOTTOM = FRONT_SPACING + "What can I do for you?";
+    public static final String BYE_HEADER = FRONT_SPACING + "Bye Bye! Hope to see you again soon!";
```

Comment:
I like that you define these static string at the beginning of your code. You used understandable names and made the content more clear.